README.md

```diff
@@ -9,52 +9,43 @@ Tackles the problem of processing asynchronous jobs in reliable manner by relyin
 Add this line to your application's Gemfile:
 
 ```ruby
-gem "tackle"
+gem "tackle", :git => "https://github.com/renderedtext/tackle"
 ```
 
 ## Usage
 
-```ruby
-require "tackle/worker"
-
-class ReliableWorker
-
-  def initialize
-    @worker = Tackle::Worker.new("test-exchange",
-                                 "test-queue", :url => "amqp://localhost:5672"
-                                               :retry_limit => 2,
-                                               :retry_delay => 15,
-                                               :logger => logger)
-  end
-
-  def run
-    @worker.subscribe { |message| perform(message) }
-  end
-
-  def perform(message)
-    # do something fun
-  end
+Subscribe consumer:
```

Comment:
Let's convert these into subtitles (with `###`). Subtitles are links, and can be shared more easily.